JS-Assignment-Operators.md

```diff
@@ -0,0 +1,74 @@
+
+## Introduction
```

Comment:
```diff
-## Introduction
+### Introduction
```

This is so it will display data up to the firs `##` in the chat, as it is right now, calling that article from the chat would only give the title and no data snippet.